### ISSUE TITLE
Handle lgtm errors\warnings\offers

### ISSUE THREAD

**Issue by Matanpugach** [TechnionYP5779/team1#144]

**Comment by yoavz1997** [comment]
What is the lgtm?

**Comment by Matanpugach** [comment]
@yoavz1997 what @yossigil sent in the telegram group. ([this](https://lgtm.com/projects/g/TechnionYP5779/team1/alerts/))

**Comment by Matanpugach** [comment]
The LGTM doesn't update on it own. In [here ](https://lgtm.com/help/lgtm/managing-pr-integration)described a way of update it using PR. Is that how we should do it? I didn't find much else online.

**Comment by ofiralexi** [comment]
there is still 1 alert. Please fix it 
![image](https://user-images.githubusercontent.com/37834992/48870144-5475c500-ede8-11e8-83c3-0886ff6393a4.png)

**Comment by Matanpugach** [comment]
Yes but it is fixed in the code! It doesn't sync! I tried again now, hope it will work. I didn't find ANY solutions. Check the code yourself - the problem is not there. @ofiralexi
